transformer_engine/pytorch/attention.py

```diff
@@ -951,6 +965,13 @@ def forward(
         use_fused_attention = (use_fused_attention
                               and is_backend_avail
                               and self.num_gqa_groups == self.num_attention_heads)
+        if (self.deterministic
+            and fused_attention_backend == FusedAttnBackend["F16_arbitrary_seqlen"]):
+            use_fused_attention = False
+            warnings.warn(
+                "Disabling usage of FusedAttention since the FusedAttention"
+                "backend does not support deterministic exection."
+            )
```

Comment:
@ksivaman hey, maybe change this to
`this FusedAttention backend does not support deterministic execution`? Sorry I didn't catch this earlier.

Reply:
Done

Reply:
Sorry I merged and also forgot about that, maybe we can squeeze that into your miscellaneous fixes MR? :-)

Reply:
@cyanguwa